packages/controller/src/resources/cortex/index.ts

```diff
@@ -332,6 +332,7 @@ export function CortexResources(
         }
       }
     },
+    purger: { enable: true },
```

Comment:
Purger is just for chunks right now so there is no point in enabling this API :-(

Reply:
> [ENHANCEMENT] Added new experimental API endpoints: POST /purger/delete_tenant and GET /purger/delete_tenant_status for deleting all tenant data. Only works with blocks storage. Compactor removes blocks that belong to user marked for deletion. #3549 #3558

(https://github.com/cortexproject/cortex/blob/master/CHANGELOG.md#master--unreleased)

! :)

Reply:
Right on!

Reply:
I imagine this needs a few new URLs to be proxied